notebooks/community/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
<p>use the previous variable endpoint to delete the endpoint.</p><p> </p><p>use the previous variable model to delete the model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
Not sure I understand, because I've been using `endpoint[-1]` for each of the cells. Are you suggesting that I store that in its own variable (e.g. `my_endpoint`) and use that everywhere instead of `endpoint[-1]`?

I've updated the "Cleaning up section" for now to:
```
1 # Retrieve ID of endpoint
2 ENDPOINT_ID = int(endpoint[-1].name)
3
4 # Undeploy model from endpoint
5 endpoint[-1].undeploy_all()
6
7 # Delete endpoint resource
8 ! gcloud ai endpoints delete $ENDPOINT_ID --quiet --region $REGION
9
10 # Delete BigQuery ML model
11 ! bq rm -f --model $PROJECT_ID\:$BQ_DATASET_NAME\.$BQML_MODEL_NAME
```